3.interpret_models/log_reg_plate_3_3p_cp_renorm_fs_data/feature_importances/nbconverted/calc.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Determine the best features using a Logistic Regression Model
+
+# ## Imports
+
+# In[1]:
+
+
+import sys
+import pandas as pd
+import numpy as np
+from pathlib import Path
+from joblib import load
+
+
+# ## Find the git root Directory
+
+# In[2]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
```

Comment:
same comment about using `__file__` metadata call

Reply:
I think I'll leave this for now. See the comment above